src/eval/BaselineEvals.py

```diff
@@ -0,0 +1,396 @@
+import argparse
+import matplotlib.pyplot as plt
+import pandas as pd
+import pickle
+import seaborn as sns
+import numpy as np
+import time
+import torch
+import os
+import sys
+import cuml
+
+from cuml.manifold import TSNE, UMAP
```

Comment:
check if cuml is installed, else use sklearn
from sklearn.manifold import TSNE
import UMAP